tests/tests/control_atm_aerosols

```diff
@@ -0,0 +1,85 @@
+###############################################################################
+#
+#  global control test with prognostic aerosols: GFSv16 FV3-GOCART at C96L127
+#
+###############################################################################
+
+export TEST_DESCR="Compare global results for prognostic aerosols with previous trunk version"
+
+# export NEW_BASELINE=/scratch1/NCEPDEV/nems/Raffaele.Montuoro/emc.nemspara/RT/GOCART/develop-20210610/INTEL/baseline
+export RTPWD=/scratch1/NCEPDEV/nems/Raffaele.Montuoro/emc.nemspara/RT/GOCART/develop-20210610/INTEL
```

Comment:
No personal directories. Please clean this up.

Reply:
Done in e4404cf7f81ad9ebde4f31858a919ea8c3934523. Thanks for catching this!